aspnetcore/signalr/publish-to-azure-web-app.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Publish an ASP.NET Core SignalR app to Azure Web App
+author: rachelappel 
+description: Publish an ASP.NET Core SignalR app to Azure Web App
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 04/20/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/publish-to-azure-web-app
+---
+
+# Publish an ASP.NET Core SignalR app to an Azure Web App
+
+[Azure Web App](https://docs.microsoft.com/azure/app-service/app-service-web-overview) is a [Microsoft cloud computing](https://azure.microsoft.com/) platform service for hosting web apps, including ASP.NET Core.
+
+## Publish the app
+
+Visual Studio provides built-in tools for publishing to an Azure Web App. Visual Studio Code user can use [Azure CLI](https://docs.microsoft.com/cli/azure) commands to publish apps to Azure. This article covers publishing using  the tools in Visual Studio. To publish an app using Azure CLI, see [Publish an ASP.NET Core app to Azure with command line tools](xref:tutorials/publish-to-azure-webapp-using-cli).
```

Comment:
- https://docs.microsoft.com/cli/azure --> /cli/azure
- Remove the extra space between "publishing using" and "the tools"